errors_errno.go

```diff
@@ -0,0 +1,22 @@
+// +build aix darwin dragonfly freebsd linux nacl nacljs netbsd openbsd solaris windows
```

Comment:
It feels like we're likely to forget to update this along the way, given how big the build tag is. Can we do something shorter? It seems like it almost builds for everything.

Reply:
It's actual list that have specific errno. For example js doesn't.

Reply:
I realise that. But wouldn't it be easier to do `!js !z390` or something?

Reply:
I don't think there is a safe way to make it shorter.
Added a comment to generate the list.